lib/Core/Executor.h

```diff
@@ -491,6 +490,8 @@ class Executor : public Interpreter {
     symPathWriter = tsw;
   }
 
+  PTree &getPTree() { return *processTree; }
```

Comment:
Getters that only return values are an antipattern. By returning a reference it becomes "public" anyway.

Reply:
Would const reference be ok?